test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Microsoft.AspNet.OData.Test.Shared.projitems

```diff
@@ -102,6 +103,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)Builder\TestModels\OrderHeader.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Builder\TestModels\OrderLine.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Builder\TestModels\PropertyAlias.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)\..\..\TestModels\RecursiveComplexTypesModels.cs">
+      <Link>Builder\TestModels\RecursiveComplexTypesModels.cs</Link>
+    </Compile>
```

Comment:
I'd prefer to create a common model project with the shared models (classes) and referred by UT and E2E?

Reply:
Ok, I'll give it a try.

Reply:
Done